### ISSUE TITLE
Suggestion: Moshi 

### ISSUE THREAD

**Issue by iNoles** [lovasoa/bad_json_parsers#24]
I like to add Moshi on the list.
https://github.com/square/moshi

**Comment by lovasoa** [comment]
Yes, please do! I'm open to pull requests. Your pull request should contain an implementation, a new line in the README, and a new test in travis.
